src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf16Utility.Validation.cs

```diff
@@ -434,5 +483,22 @@ static Utf16Utility()
             scalarCountAdjustment = tempScalarCountAdjustment;
             return pInputBuffer;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int Arm64MoveMask(Vector128<byte> value)
```

Comment:
@kunalspathak  's [suggestions](https://github.com/dotnet/runtime/pull/38653#discussion_r452562247) in my other PR probably applies for this method as well: avoid the static variables since accessing those is less efficient than passing them as arguments, then simplify the method content itself to only do one `AddPairwise`.